### DISCUSSION TITLE
Livewire navigate

### DISCUSSION THREAD

**Discussion by ali-hasnain-dev** [livewire/livewire#6711] [1. Help]
Hello community,

I need some helps for this problems. I'm using livewire navigate feature in admin panel but when i change navigation my navbar does not work then I've to click once more on navigate then it works but then if i change navigation problem again occured as shown in video
[Uploading scrnli_9_7_2023_11-29-54 PM.webm…]()

**Comment by joshhanley** [comment]
@alihasnain01 your video didn't finish uploading before you submitted, so it's currently not working.

**Reply by ali-hasnain-dev** [reply]
Sorry @joshhanley I don't know why this is not uploded, here is video 
I've to click twice so navbar work properly

[scrnli_9_7_2023_11-29-54 PM.webm](https://github.com/livewire/livewire/assets/143272549/1fce6503-b07f-4c89-b7a6-99bb2d8df4e2)

**Reply by luanfreitasdev** [reply]
maybe related

https://github.com/livewire/livewire/discussions/6448#discussioncomment-6924902
